release_bot/releasebot.py

```diff
@@ -40,18 +40,55 @@ def __init__(self, configuration):
         self.fedora = Fedora(configuration)
         self.logger = configuration.logger
         self.new_release = {}
+        self.new_pr = {}
 
     def cleanup(self):
         if 'tempdir' in self.new_release:
             self.new_release['tempdir'].cleanup()
         self.new_release = {}
+        self.new_pr = {}
         self.github.comment = []
         self.fedora.progress_log = []
 
     def load_release_conf(self):
         # load release configuration from release-conf.yaml in repository
-        release_conf = self.conf.load_release_conf(self.new_release['fs_path'])
+        repo = self.github.clone_repository()
+        release_conf = self.conf.load_release_conf(repo.repo_path)
         self.new_release.update(release_conf)
+        repo.cleanup()
```

Comment:
Just wondering whether it's not an overkill to clone whole repo to get one tiny config.
Can we at least clone it with `--depth 1` in this case ?